stable/concourse/templates/secrets.yaml

```diff
@@ -53,4 +53,7 @@ data:
   aws-ssm-session-token: {{ .Values.secrets.awsSsmSessionToken | b64enc | quote }}
   {{- end }}
   {{- end }}
+  {{- if .Values.web.metrics.influxdb.enabled }}
+  influxdb-password: {{ default "" .Values.secrets.influxdbPassword | b64enc | quote }}
```

Comment:
Add empty default value to `values.yaml` instead of using the `default` function.

Reply:
why?

Reply:
OK, accepted. Let's keep it this way for consistency. Generally, we advice against using the `default` function if not necessary.